master/docs/manual/cfg-wwwhooks.rst

```diff
@@ -135,7 +135,8 @@ The simplest way to use GitHub hook is as follows:
 
 .. code-block:: python
 
-    c['www'] = dict(...,
+    c['www'] = dict(    
+        'base': True
```

Comment:
why? no need to have base!

Reply:
it didn't work without it.

Reply:
what didn't work? What is the symptom?

c['www']['base'] is not an available config
c['www']['change_hook_dialects']['base'] is, but only enables  the base hook. 

The github hook should not be dependent on the base hook. I double checked on the code that there is no code smell in that direction. Please double check on your side that this is not a confusion.

Reply:
my confusion, I reverted back to PyPi release and it's ok. I must have tested (what I thought was) the endpoint using http://cam1.local:8010/change_hook/

Reply:
can you fix your patch then?